substrate/frame/revive/fixtures/build.rs

```diff
@@ -207,6 +202,17 @@ mod build {
 		invoke_build(tmp_dir_path)?;
 
 		write_output(tmp_dir_path, &out_dir, entries)?;
+
+		#[cfg(unix)]
+		{
+			let symlink_dir: PathBuf = env::var("CARGO_WORKSPACE_ROOT_DIR")?.into();
```

Comment:
I thinkk this does only work on our CI? Shouldn't this be something like `if let Ok(symlink_dir) = ..` instead of the `#[cfg(unix)]`? Could probably also use a comment why this is needed, it's not obvious to me :)